### ISSUE TITLE
A commit was lost (evil-insert in git commit buffers)

### ISSUE THREAD

**Issue by ruimams** [hlissner/doom-emacs#300]
Hi @hlissner, I'm not sure if you are aware but 8928ff07d7d7d65f77afb652d749cf4d9dc83bb5 is gone (not sure what term to use here) from the repo, despite still showing up on github. It was, during some time, on the `develop` branch but it isn't anymore, neither on `develop` nor `master` branches.

I guess this happened with some force push, or something like that. If you are aware of this feel free to close the issue.

Could you please bring the commit back? I really want evil-mode on git commit buffers. 😅

Thanks! 👍

**Comment by hlissner** [comment]
Ah! My bad. A struggled to find a dangling commit when I was merging 2.0.8 and never found it, so I took the lazy way out. This was it! 

I shouldn't be merging that late in the evening!

Anyhow, it's back in `develop` now, for the time being. I've noticed a laundry list of bugs and outdated documentation I'd like to address in the next week or two, so I'll fit this into a 2.0.9 release right after the holidays. I'll leave this open until then.
